src/operator/quantization/quantized_fully_connected.cc

```diff
@@ -72,7 +79,7 @@ bool QuantizedFullyConnectedType(const nnvm::NodeAttrs& attrs,
   CHECK_EQ(in_type->size(), num_inputs * 3);
   CHECK_EQ(out_type->size(), 3U);
 
-  for (size_t i = 0; i < num_inputs; ++i) {
+  for (size_t i = 1; i < num_inputs; ++i) {
```

Comment:
Why skip `i = 0`?

Reply:
input[0] will supports both INT8 and UINT8 here.

Reply:
Then please add check for input[0] on INT8 or UINT8.